ui/server/src/main/scala/pl/touk/nussknacker/ui/config/processtoolbar/ToolbarCondition.scala

```diff
@@ -0,0 +1,18 @@
+package pl.touk.nussknacker.ui.config.processtoolbar
+
+import pl.touk.nussknacker.ui.config.processtoolbar.ToolbarConditionType.ToolbarConditionType
+
+object ToolbarConditionType extends Enumeration {
+
+  type ToolbarConditionType = Value
+
+  val OneOf: Value = Value("oneof")
+  val AllOf: Value = Value("allof")
```

Comment:
do we actually need it? :)

Reply:
Right now now.. :) But I think I will use it after decision about disabled/hide param.